person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unkown', parents_permission = `true`)
```

Comment:
- [ ] Nice adding the required changes however you shouldn't put backticks around the true keyword that will return an empty string so please try to remove the backticks (single quotes )